R/calendar_write_auth.R

```diff
@@ -0,0 +1,39 @@
+#' Authorize Google Calendar with write permissions
+#'
+#' This is a convenience function as an alternative to \code{calendar_auth(scopes = "https://www.googleapis.com/auth/calendar.events")}
+#'
+#' @inheritParams calendar_auth
+#' @export
+calendar_write_auth <- function(
```

Comment:
I like the idea, but this function should be simplified to a simple wrapper around `calendar_auth()` where you simply pass the modified scope into `calendar_auth()`.  This will reduce code duplication.

Reply:
Removing in favor of manually specifying scope in the original `calendar_auth()` function.